### PR TITLE
Initial version of census upgrade bot

### PR DESCRIPTION
**Pull request by rwestberg**
Allows automated conversion of census.xml to the Skara format.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

### Reviewers
 * [Erik Helin](https://openjdk.java.net/census#ehelin) (@edvbld - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/skara pull/1019/head:pull/1019`
`$ git checkout pull/1019`

### CODE REVIEW

File: xml/build.gradle [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * questions.
+ */
+
+module {
+    name = 'org.openjdk.skara.xml'
+}
+
+publishing {
+    publications {
+        webrev(MavenPublication) {
```
Comment (edvbld) [inline xml/build.gradle:30]:
```suggestion
        xml(MavenPublication) {
```

**Review by edvbld — COMMENTED** [review]
Looks good, but may I suggest renaming `CensusSyncDowngradeBot.java` to `CensusSyncUnifyBot.java` and `CensusSyncUpgradeBot.java` to `CensusSyncSplitBot.java`? Makes it a bit more clear what the bots are doing :smiley:

**Review by edvbld — APPROVED** [review]
Looks great!

File: bots/censussync/src/main/java/org/openjdk/skara/bots/censussync/CensusSyncSplitBot.java [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+                        reviewers.add(username);
+                        break;
+                    case "committer":
+                        committers.add(username);
+                        break;
+                    case "author":
+                        authors.add(username);
+                        break;
+                    default:
+                        if (name.equals("openjfx") && (username.equals("dwookey") || username.equals("jpereda"))) {
```
Comment (magicus) [inline bots/censussync/src/main/java/org/openjdk/skara/bots/censussync/CensusSyncSplitBot.java:179]: I know this is already integrated, but these sounds like weird hard-coded exceptions? If there are issues in the census data, maybe they should be fixed instead?

**Pull request closed without merging**